### ISSUE TITLE
Trying to set an emoji as status crashes the app

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/android#8479]
### Steps to reproduce
1. Open the app
2. Switch account
3. Try to set a status
4. Click on the emoji button

### Expected behaviour
I can set a status

### Actual behaviour
```
************ CAUSE OF ERROR ************

java.lang.NullPointerException
	at com.nextcloud.ui.ChooseAccountDialogFragment.getBinding(ChooseAccountDialogFragment.kt:71)
	at com.nextcloud.ui.ChooseAccountDialogFragment.avatarGenerated(ChooseAccountDialogFragment.kt:196)
	at com.owncloud.android.datamodel.ThumbnailsCacheManager$AvatarGenerationTask.onPostExecute(ThumbnailsCacheManager.java:921)
	at com.owncloud.android.datamodel.ThumbnailsCacheManager$AvatarGenerationTask.onPostExecute(ThumbnailsCacheManager.java:868)
	at android.os.AsyncTask.finish(AsyncTask.java:755)
	at android.os.AsyncTask.access$900(AsyncTask.java:192)
	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:772)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7551)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:995)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30160090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Redmi
Device: begonia
Model: Redmi Note 8 Pro
Id: QP1A.190711.020
Product: begonia

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: V12.0.7.0.QGGMIXM
```

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version: 10

Device model: Xiaomi Redmi Note 8

Stock or customized system: Xiaomi stock

Nextcloud app version: playstore version

Nextcloud server version: 21.0.2

Reverse proxy: no

**Comment by nickvergessen** [comment]
After trying it again I know see my previous status. On the crashed attempt I might have been too fast as I saw the default smilie and an empty status message

**Comment by AndyScherzinger** [comment]
> On the crashed attempt I might have been too fast as I saw the default smilie and an empty status message

@nickvergessen when hitting the button where all avatars of the account already loaded. Looking at the stack trace my guess is the view closed before all avatars where loaded and the async task tried to set an avatar on a already destroyed view.

**Comment by nickvergessen** [comment]
Might also be as my local instance was turned of as I tried to set the status before I started my dev maschine with the server

**Comment by AndyScherzinger** [comment]
Fixed via #8356

**Comment by AndyScherzinger** [comment]
Duplicate of #8375
